smoketest/configs/basic-vyos

```diff
@@ -89,6 +89,14 @@ service {
                     start 192.168.0.20
                     stop 192.168.0.240
                 }
+                static-mapping TEST1 {
```

Comment:
This is a testcase for duplicate MACs with different IPs, can we also have a case for duplicate IPs with different MACs?